packages/cli/src/commands/stream/state.ts

```diff
@@ -0,0 +1,26 @@
+import { Command } from '../../command'
+import type { CommandFlags } from '../../command'
+
+export default class State extends Command<CommandFlags, { streamId: string }> {
+  static description = 'get the state of a Stream'
+
+  static args = [
+    {
+      name: 'streamId',
+      required: true,
+      description: 'ID of the Stream',
+    },
+  ]
+
+  async run(): Promise<void> {
+    this.spinner.start(`Querying stream ${this.args.streamId}`)
+
+    try {
+      const stream = await this.ceramic.loadStream(this.args.streamId)
+      this.spinner.succeed(`Successfully queried stream ${this.args.streamId}`)
+      this.logJSON(stream)
```

Comment:
does this actually log the right thing?  I'd have expected you to have to log `stream.state`

Reply:
cc @Sterahi

Reply:
Good catch, thanks.